packages/instant/src/util/error_description.ts

```diff
@@ -0,0 +1,23 @@
+import { AssetBuyerError } from '@0xproject/asset-buyer';
+
+import { bestNameForAsset } from '../util/asset_data';
+
+const humanReadableMessageForError = (error: Error, assetData?: string): string | undefined => {
+    if (error.message === AssetBuyerError.InsufficientAssetLiquidity) {
+        const assetName = bestNameForAsset(assetData, 'of this asset');
+        return `Not enough ${assetName} available`;
+    }
+
+    return undefined;
+};
+
+export const errorDescription = (error?: any, assetData?: string): { icon: string; message: string } => {
+    let bestMessage: string | undefined;
+    if (error instanceof Error) {
+        bestMessage = humanReadableMessageForError(error, assetData);
+    }
+    return {
+        icon: '😢',
+        message: bestMessage || 'Something went wrong...',
+    };
+};
```

Comment:
I would perhaps create one `util/error.ts` file that has one `error` or `errorUtil` object export and also provides the `flashNewError` utility?

Reply:
I'll try this and see how it feels once implemented. `ErrorFlasher` is kinda big though, not sure if I love it being grouped together with other things

Reply:
Done in d45ffb33513057789a2e1cf03a9f01ee05846e17